test/unit/key_def.cc

```diff
@@ -457,16 +460,875 @@ test_tuple_validate_key_parts_raw(void)
 	check_plan();
 }
 
+static bool
+key_def_field_is_nullable(const struct key_def *key_def, uint32_t fieldno)
+{
+	for (uint32_t i = 0; i < key_def->part_count; i++) {
+		if (key_def->parts[i].fieldno == fieldno) {
+			if (key_part_is_nullable(&key_def->parts[i]))
+				return true;
+			else
+				return false;
+		}
+	}
+	return false;
+}
+
+static bool
+key_def_field_is_indexed(const struct key_def *key_def, uint32_t fieldno)
+{
+	for (uint32_t i = 0; i < key_def->part_count; i++)
+		if (key_def->parts[i].fieldno == fieldno)
+			return true;
+	return false;
+}
+
+static void
+test_create_common_cases(std::vector<struct tuple *> &tuples_eq,
```

Comment:
I'd call it like `test_generate_common_cases`. Also, it would be nice to cover it with a comment - this function inspects passed `key_def` and generates tuples to test it, but it does not generate tuples for `key_def`'s of unique indexes because of inconsistent behavior and so on.

Reply:
Renamed and commented.